### DISCUSSION TITLE
Delta for humidity, temperature, CO2

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#20353] [Ideas]
I have several sensor in my house to monitor air quality. To have fast updates I have set teleperiod to 10, but this generates a lot of mqtt messages of which are mostly duplicate.

I would like to make a pull request so a delta for temperature, humidity and CO2 can be configured. When the sensor value is beyond the delta range the sensor data is send immediately instead of waiting for the next teleperiod.

This would make the MQTT traffic a lot less but still have fast or even faster updates on sensor changes.

**Comment by sfromis** [comment]
There already exists a rules example for sensor deltas, and even with a faster reaction time than TelePeriod.
https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached

**Comment by jeroenst** [comment]
That's nice, and would work for me. But wouldn't it be a good idea when I integrate commands like:
temperaturedelta, humiditydelta and co2delta like powerdelta?
